docs/user/03-traces.md

```diff
@@ -502,3 +502,21 @@ System-related spans reported by Istio are filtered out without the opt-out opti
           - name: "kyma-traces"
           randomSamplingPercentage: 60
     ```
+
+### Buffer Filling Up
+
+  Cause: The backend sizing is inadequate
+
+  Remedy:
+  
+  - Option 1: Increase ingestion rate capabilities in your backend (i.e. scale out SAP Cloud Logging or Dynatrace instances)
```

Comment:
I would remove Dynatrace here. The possibility to scale here might be limited to users who run their own on-prem cluster (which should be the exception) and more confusing for anyone else.